### PR TITLE
fix: add support for bool values in time series result

### PR DESCRIPTION
**Pull request by makeavish**

### CODE REVIEW

File: pkg/query-service/app/clickhouseReader/reader.go [hunk 1]
```diff
@@ -4096,8 +4096,19 @@ func readRow(vars []interface{}, columnNames []string) ([]string, map[string]str
 				groupBy = append(groupBy, fmt.Sprintf("%v", reflect.ValueOf(v).Elem().Int()))
 				groupAttributes[colName] = fmt.Sprintf("%v", reflect.ValueOf(v).Elem().Int())
 			}
+		case *bool:
+			if _, ok := constants.ReservedColumnTargetAliases[colName]; ok {
+				if *v {
+					point.Value = 1
+				} else {
+					point.Value = 0
+				}
```
Comment (srikanthccv) [inline pkg/query-service/app/clickhouseReader/reader.go:4105]: How can a boolean be target column?

Comment (makeavish) [reply pkg/query-service/app/clickhouseReader/reader.go:4105]: When we groupBy over a boolean column.

Comment (srikanthccv) [reply pkg/query-service/app/clickhouseReader/reader.go:4105]: Aggregation on the attributes such as sum, max etc are the target columns. Group by is only for aggregation among the groups but group by column itself is not target column.

**Review by srikanthccv — APPROVED** [review]

File: pkg/query-service/app/clickhouseReader/reader.go (hunk 1 above)
Comment (makeavish) [reply pkg/query-service/app/clickhouseReader/reader.go]: Understood that you are talking about the reserved target column. Pushed the changes.

**Pull request merged**